controllers/build/image_stream_kogito.go

```diff
@@ -37,59 +37,39 @@ const (
 var (
 	// imageStreamDefaultAnnotations lists the default annotations for ImageStreams
 	imageStreamDefaultAnnotations = map[string]map[string]string{
-		infrastructure.KogitoQuarkusUbi8Image: {
+		infrastructure.KogitoRuntimeNative: {
 			"openshift.io/provider-display-name": "KIE Group",
 			"openshift.io/display-name":          "Runtime image for Kogito based on Quarkus native image",
 		},
-		infrastructure.KogitoQuarkusJVMUbi8Image: {
+		infrastructure.KogitoRuntimeJVM: {
 			"openshift.io/provider-display-name": "KIE Group",
-			"openshift.io/display-name":          "Runtime image for Kogito based on Quarkus JVM image",
+			"openshift.io/display-name":          "Runtime image for Kogito based JVM image",
 		},
-		infrastructure.KogitoQuarkusUbi8s2iImage: {
+		infrastructure.KogitoBuilderImage: {
 			"openshift.io/provider-display-name": "KIE Group",
-			"openshift.io/display-name":          "Platform for building Kogito based on Quarkus",
-		},
-		infrastructure.KogitoSpringBootUbi8Image: {
-			"openshift.io/provider-display-name": "KIE Group",
-			"openshift.io/display-name":          "Runtime image for Kogito based on SpringBoot",
-		},
-		infrastructure.KogitoSpringBootUbi8s2iImage: {
-			"openshift.io/provider-display-name": "KIE Group",
-			"openshift.io/display-name":          "Platform for building Kogito based on SpringBoot",
+			"openshift.io/display-name":          "Platform for building Kogito based on Quarkus or Spring Boot",
```

Comment:
Nitpick (also in other places):
```suggestion
			"openshift.io/display-name":          "Platform for building Kogito based on Quarkus or SpringBoot",
```

Reply:
or maybe not...
I always thought it is written as one word

Reply:
yeah on the images I was asked to keep them separate

Reply:
That's right: https://spring.io/projects/spring-boot